pkg/download/document/download.go

```diff
@@ -32,17 +33,34 @@ import (
 	v2 "github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/project/v2"
 )
 
+var documentMapping = map[string]config.DocumentKind{
+	documents.Dashboard: config.DashboardKind,
+	documents.Notebook:  config.NotebookKind,
+	documents.Launchpad: config.LaunchpadKind,
+}
+
 func Download(client client.DocumentClient, projectName string) (v2.ConfigsPerType, error) {
-	result := make(v2.ConfigsPerType)
+	// due to the current test setup, the types must be downloaded in order. This should be changed eventually
+	var typesToDownload = []documents.DocumentType{
```

Comment:
could this just be replaced with `slices.Collect(maps.Values(documentMapping))`?

Reply:
Tried it, but the issue with maps is that the order is by design not specified. And as the comment states, the test setup is at fault that it has to be in a certain order :(